### PR TITLE
Fix compatibility with AS Concern

### PR DESCRIPTION
**Pull request by tycooon**
Fixes the following exception when using Memery in a `ActiveSupport::Concern` module:

```
ArgumentError:
  wrong number of arguments (given 0, expected 1)
# ./lib/memery.rb:15:in `included'
```

### CODE REVIEW

**Comment by tycooon** [comment]
@AlexWayfer what do you think? It's pretty common to use both AS::Concern and Memery in the same module, so I think that we should better drop that auto-include feature.

**Comment by AlexWayfer** [comment]
Done. You can merge #27 into this branch and then merge this PR if everything is OK.

File: lib/memery.rb [hunk 1]
```diff
@@ -32,16 +27,14 @@ def memoized?(method_name)
       return false unless defined?(@_memery_module)
 
       @_memery_module.method_defined?(method_name) ||
-        @_memery_module.private_method_defined?(method_name)
```
[the remark this answers is not in the document]
Comment (tycooon) [reply lib/memery.rb]: Yep, but I'm usually OK with some minimal cosmetic changes :)

[the remark this answers is not in the document]
Comment (AlexWayfer) [reply lib/memery.rb]: OK, but let's add the RuboCop to the CI of this repository and check if it's OK.

Can we switch from Travis CI? I have no permissions to do this, and I suggest Cirrus CI or Circle CI (a config of the first one will be shorten).

[the remark this answers is not in the document]
Comment (tycooon) [reply lib/memery.rb]: There actually is rubocop in the CI. It gets called in the `bundle exec rake` command.

And btw what's the problem with Travis?

[the remark this answers is not in the document]
Comment (AlexWayfer) [reply lib/memery.rb]: > There actually is rubocop in the CI. It gets called in the bundle exec rake command.

Oh, OK, then it's strange that both approaches (with indentation and without) are acceptable.

> And btw what's the problem with Travis?

I had many issues with environment, but if it satisfies you — never mind. Also, it uses the deprecated method of integration with GitHub, as I know.

Also: …

image

**Pull request merged**